### PR TITLE
Type conversions docs

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

- documenting changes from https://github.com/spatialos/gdk-for-unity/pull/1063/

#### Tests

- improbadoc serve

#### Documentation

- yes this is docs

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/modules/transform-sync/utilities.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<%(TOC max="4")%>
+
+# Transform utilities
+
+The transform synchronization module includes a [`TransformUtils`]({{urlRoot}}/api/transform-synchronization/transform-utils) class, populated with a set of static methods that make it nicer to use the feature module.
```
Comment (jessicafalk) [inline docs/modules/transform-sync/utilities.md]:
```suggestion
The transform synchronization module includes a [`TransformUtils`]({{urlRoot}}/api/transform-synchronization/transform-utils) class, populated with a set of static methods that makes it nicer to use the feature module.
```

File: docs/modules/transform-sync/utilities.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| `Vector3`              | `.ToCoordinates()`          | `Coordinates`          |
+| `Vector3`              | `.ToFixedPointVector3()`    | `FixedPointVector3`    |
+| `FixedPointVector3`    | `.ToUnityVector()`          | `Vector3`              |
+| `FixedPointVector3`    | `.ToCoordinates()`          | `Coordinates`          |
+| `CompressedQuaternion` | `.ToUnityQuaternion()`      | `Quaternion`           |
+| `Quaternion`           | `.ToCompressedQuaternion()` | `CompressedQuaternion` |
+
+## Snapshot constructor
+
+The `CreateTransformSnapshot` method takes can construct `TransformInternal` snapshot, given uncompressed position, rotation or velocity as arguments.
```
Comment (jessicafalk) [inline docs/modules/transform-sync/utilities.md]:
```suggestion
The `CreateTransformSnapshot` method constructs a `TransformInternal` snapshot, given uncompressed position, rotation or velocity as arguments.
```

File: docs/reference/concepts/code-generation.md [hunk 3]
```diff
@@ -10,32 +10,51 @@ The code generator runs when you do one of the following:
...
-| `int32` / `sint32` / `fixed32` | `int`                   |
-| `uint32`                       | `uint`                  |
-| `int64` / `sint64`/ `fixed64`  | `long`                  |
-| `uint64`                       | `ulong`                 |
-| `float`                        | `float`                 |
-| `bool`                         | `BlittableBool`         |
-| `string`                       | `string`                |
-| `bytes`                        | `byte[]`                |
-| `EntityId`                     | `Improbable.Gdk.Core.EntityId` |
+| Schemalang type                |               C# type                |
```
Comment (jessicafalk) [inline docs/reference/concepts/code-generation.md]: nice! 🎉

File: docs/reference/concepts/code-generation.md [hunk 4]
```diff
@@ -10,32 +10,51 @@ The code generator runs when you do one of the following:
...
+|-----------------------|:---------------------------------------------:|
+| `map<K, V>`           | `System.Collections.Generic.Dictionary<K, V>` |
+| `list<T>`             |     `System.Collections.Generic.List<T>`      |
+| `option<T>`           |        `Improbable.Gdk.Core.Option<T>`        |
+
+### Standard schema library types
+
+The code generator generates a C# struct for each type defined in the [standard schema library](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/schema/standard-schema-library). The generated struct is annotated with the [`System.Serializable` attribute](https://docs.unity3d.com/ScriptReference/Serializable.html) and has a constructor with a parameter per schema field.
+
+The GDK also implements the `+`, `-`, `*`, `/`, `==` and `!=` operators for the `Coordinates` and `EdgeLength` structs. In addition, the structs implement methods for converting to and from the Unity `Vector3` type.
```
Comment (jessicafalk) [inline docs/reference/concepts/code-generation.md]:
```suggestion
The GDK also implements the `+`, `-`, `*`, `/`, `==` and `!=` operators for the `Coordinates` and `EdgeLength` structs. In addition, the structs implement methods for converting their data to and from the Unity `Vector3` type.
```

File: docs/reference/concepts/code-generation.md (hunk 3 above)
Comment (paulbalaji) [reply docs/reference/concepts/code-generation.md]: I'd recommend this vscode plugin: https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter

File: docs/modules/transform-sync/utilities.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+## Type conversion methods
+
+The following static methods can be used to construct a `FixedPointVector3` or `CompressedQuaternion` from their respective native Unity types.
+
+| Static Method                                            | Result Type            |
+|----------------------------------------------------------|------------------------|
+| `FixedPointVector3.FromUnityVector(Vector3 v)`           | `FixedPointVector3`    |
+| `CompressedQuaternion.FromUnityQuaternion(Quaternion q)` | `CompressedQuaternion` |
+
+The methods below can be called to easily convert a variable to and from native Unity types.
```
Comment (jamiebrynes7) [inline docs/modules/transform-sync/utilities.md]:
```suggestion
The extension methods below can be called to easily convert a variable to and from native Unity types.
```

Maybe?

File: docs/toc.md [hunk 6]
```diff
@@ -61,6 +61,7 @@
         - [How it works]({{urlRoot}}/modules/transform-sync/how-it-works)
         - [Set up transform synchronization]({{urlRoot}}/modules/transform-sync/setup-transform)
         - [Strategies]({{urlRoot}}/modules/transform-sync/strategies)
+        - [Transform utils]({{urlRoot}}/modules/transform-sync/utilities)
```
Comment (jamiebrynes7) [inline docs/toc.md]:
```suggestion
        - [Transform utilities]({{urlRoot}}/modules/transform-sync/utilities)
```

Prefer full names in the toc

File: docs/modules/transform-sync/utilities.md (hunk 5 above)
Comment (paulbalaji) [reply docs/modules/transform-sync/utilities.md]: I nearly wrote that, but technically not every one of them is an extension method

**Pull request merged**